js-exercises/alternating-characters/alternatingCharacters.js

```diff
@@ -0,0 +1,19 @@
+
+function alternatingCharacters(arr) {
+    let flag = 0
+    var noOfCharactersToDelete = arr.map(stringToConsider =>{
+      if(stringToConsider.length>1)
+      {
+        flag = 0
+     for(let i = 0 ; i<stringToConsider.length-1;i++)
```

Comment:
ES Lint points out an error for using ++ as an operator. [ESLint NoPlusPlus](https://eslint.org/docs/rules/no-plusplus)